TalentLMSLib/lib/TalentLMS/ApiRequestor.php

```diff
@@ -119,6 +119,16 @@ protected static function _handleCurlError($errno, $message)
     
     protected static function _handleApiError($rbody, $rcode, $response)
     {
+        if(isset($response['error'])){
+            $msg =
+                '<p><strong>'
+                . esc_html__(esc_html($response['error']['message']), 'talentlms')
```

Comment:
I think here, you are coupling wordpress functions to TalentLMS Lib..which must be totally independent. Any future update into TalentLMS wrapper could 'break' it (not following your rules here). It is better to return the message and read it inside wordpress and escape it there.

Reply:
Right! I removed the relevant wp escaping code and added it to the display function in errors. You will also notice I removed one escape call, since both are the same and we also use wp_kses, so it seemed redundant. Lmk if you think it has value to add the other escape there